client/dashboard/app/queries/site-backups.ts

```diff
@@ -1,9 +1,16 @@
 import { queryOptions } from '@tanstack/react-query';
 import { fetchSiteRewindableActivityLog } from '../../data/site-activity-log';
+import { fetchSiteBackups } from '../../data/site-backups';
 
 export const siteLastBackupQuery = ( siteId: number ) =>
 	queryOptions( {
 		queryKey: [ 'site', siteId, 'backups', 'last' ],
 		queryFn: () => fetchSiteRewindableActivityLog( siteId, { number: 1 } ),
 		select: ( data ) => data.current?.orderedItems[ 0 ] ?? null,
 	} );
+
+export const siteBackupsQuery = ( siteId: number ) =>
+	queryOptions( {
+		queryKey: [ 'site', siteId, 'rewind', 'backups' ],
```

Comment:
I think, we don't need `'rewind'` in the key because the query and app layers don't have this concept 🤔 (we don't have to 100% mimic the endpoint paths in the query layer)

```suggestion
		queryKey: [ 'site', siteId, 'backups' ],
```

Reply:
Makes sense. Thanks!

Addressed in https://github.com/Automattic/wp-calypso/pull/105217 -> https://github.com/Automattic/wp-calypso/pull/105217/commits/059f290730ab4dccece17d0baa4d4e24d27bf675